index.html

```diff
@@ -601,6 +601,402 @@ <h6 class="float-left">START YOUR DAY WITH OUR TOP 5 ARTICLES</h6>
+                    </div>
+                    <div class="py-3 border-top">
+                        <a href="#"><img class="float-start w-50 my-auto pe-2" src="./img/us-pic-thumb.webp" alt="homeless"></a>                
+                        <strong><a href="#" class="text-decoration-none text-dark small">Newsweek Announces Launch of the Newsweek Expert Forum </a></strong>
+                        <div class="clearfix"></div>
+                    </div>
+                </div>
+            </div>
+            
+        </section>
+        <section id="subscribe-newsletter">
+            <div class="px-3 d-flex flex-column flex-lg-row">
+                <div class="subscribe col-lg-8 py-4">
+                    <strong class="text-danger pb-4">SUBSCRIBE</strong>
+                    <div class="bg-light jumbotron mt-3 py-5">
+                        <div class="d-flex flex-column align-items-center flex-md-row align-items-md-end">
+                            <div class="position-relative p-3">
+                                <img class="" src="./img/cover-magazine.webp" alt="image" width="170" height="230">
+                                <img class="position-absolute" src="./img/cover-magazine.webp" alt="image" width="170" height="230" style="top: 30px; left: 80px;">
+                                <div style="width: 250px;"></div>
+                            </div>
+                            <div class="text-box d-flex flex-column align-items-center align-items-md-start p-4">
+                                <strong class="fs-4">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</strong>
+                                <p>All Access + Weekly Delivery.
+                                    Choose A Membership That's Perfect for You!</p>
+                                <button class="btn bg-danger text-white fw-bold">SUBSCRIBE <i class="fas fa-chevron-right"></i></button>
+                            </div>                            
+                        </div>
+                    </div>
+                </div>
+                <div class="newsletter col-lg-4 py-4">
+                    <strong class="text-danger">NEWSLETTER</strong>
+                    <div class="row d-flex flex-column flex-md-row my-3 mx-0 h-100 border">
+                        <div class="col-sm col-lg-4 p-0" style="background-image: url(./img/starting5-homepage.png); background-repeat: no-repeat; background-size: cover; background-position: center; min-height: 110px;"></div>
+                        <div class="col-sm col-lg-8 p-4 d-flex flex-column align-items-center">
+                            <strong class="fs-4">The Starting 5</strong>
+                            <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                            <div class="input-group"> 
+                                <input type="text" class="form-control" placeholder="Email address"> 
```

Comment:
Input type for newsletter must be of type "email", so you have automatic validation and correct semantic writing.